lib/Dialect/FIRRTL/Ops.cpp

```diff
@@ -710,11 +710,19 @@ static LogicalResult verifyConnectOp(ConnectOp connect) {
   if (destType.isa<AnalogType>() || srcType.isa<AnalogType>())
     return connect.emitError("analog types may not be connected");
 
+  // Destination and source types must be equivalent.
   if (!areTypesEquivalent(destType, srcType))
     return connect.emitError("type mismatch between destination ")
            << destType << " and source " << srcType;
 
-  // TODO(mikeurbach): verify source bitwidth is >= destination bitwidth.
+  // Destination bitwidth must be greater than or equal to source bitwidth.
+  int32_t destWidth = destType.getBitWidthOrSentinel();
+  int32_t srcWidth = srcType.getBitWidthOrSentinel();
+  if (destWidth > -1 && srcWidth > -1 && destWidth < srcWidth)
```

Comment:
I am only enforcing the property that `destWidth < srcWidth` for ground types that both have their width defined.

I considered erroring out for ground types that have unknown width, and concluded it would be best to allow this in the verifier, in case we want to add width inference like the SFC has. If there should be some logic here to fail on unknown types, I can add it. 

I also considered what to do about aggregate types, and concluded the verifier should just ignore the element type's widths. After a pass lowers the aggregates to ground types, the verifier can then work with the lowered IR.